modules/core/src/viewports/viewport.js

```diff
@@ -126,7 +126,8 @@ export default class Viewport {
     const [x0, y0, z0 = 0] = xyz;
 
     const [X, Y] = this.projectFlat([x0, y0]);
-    const coord = worldToPixels([X, Y, z0], this.pixelProjectionMatrix);
+    const Z = z0 * this.distanceScales.pixelsPerMeter[2];
```

Comment:
Are we interpreting x,y in current pixel distances, but Z in meters? Is that the most intuitive definition?